### DISCUSSION TITLE
Extracting mesh data

### DISCUSSION THREAD

**Discussion by eflehder1** [leap71/PicoGK#50] [Coding for (Computational) Engineers]
Hi @leap71 how could I get the number of faces in a mesh as well as the number of edges using PicoGK? I found a method .nVertexCount(), but none for the other two. 

Many thanks,
Eric

**Comment by LinKayser** [comment]
Hi @eflehder1

Number of triangles can be queried using nTriangleCount. Since each triangle has three edges, but the edges are shared between the adjacent triangles, the number of edges should be the same as the number triangles (unless I am thinking wrong!).

Best,
Lin

**Reply by eflehder1** [reply]
Hi @LinKayser 
Thank you for this!
Best,
Eric
